tests/tensor_search/test_api.py

```diff
@@ -90,3 +90,60 @@ def test_add_or_replace_documents_fields_undefined_query_param(self):
             )
             self.assertEqual(response.status_code, 400)
             mock_add_documents.assert_not_called()
+    
+    def test_add_or_replace_documents_defaults(self):
```

Comment:
is it also worth testing that manually setting `auto_refresh=True` and `auto_refresh=False`, propagates this value in the expected way?

Reply:
yep, will update this.